classes/02class/exercises/c02-network07/carloshz4/README.md

```diff
@@ -0,0 +1,127 @@
+|                           CreateRouteTable                            |
++-----------------------------------------------------------------------+
+||                             RouteTable                              ||
+|+---------------+--------------------------+--------------------------+|
+||    OwnerId    |      RouteTableId        |          VpcId           ||
+|+---------------+--------------------------+--------------------------+|
+||  016454647794 |  rtb-088d967af4f789ea7   |  vpc-0f80827681fb041d0   ||
+|+---------------+--------------------------+--------------------------+|
+|||                              Routes                               |||
+||+-----------------------+------------+--------------------+---------+||
+||| DestinationCidrBlock  | GatewayId  |      Origin        |  State  |||
+||+-----------------------+------------+--------------------+---------+||
+|||  10.10.0.0/16         |  local     |  CreateRouteTable  |  active |||
+||+-----------------------+------------+--------------------+---------+||
+
+aws ec2 create-tags \
+--resources rtb-088d967af4f789ea7  \
+--region ap-southeast-2  \
+--tags Key=Name,Value="devopsacademy-rt-public"
+```
+
+Private
+
+```
+aws ec2 create-route-table --vpc-id vpc-0f80827681fb041d0 \
+--region ap-southeast-2 
+
+-------------------------------------------------------------------------
+|                           CreateRouteTable                            |
++-----------------------------------------------------------------------+
+||                             RouteTable                              ||
+|+---------------+--------------------------+--------------------------+|
+||    OwnerId    |      RouteTableId        |          VpcId           ||
+|+---------------+--------------------------+--------------------------+|
+||  016454647794 |  rtb-09e96ac9c646cc937   |  vpc-0f80827681fb041d0   ||
+|+---------------+--------------------------+--------------------------+|
+|||                              Routes                               |||
+||+-----------------------+------------+--------------------+---------+||
+||| DestinationCidrBlock  | GatewayId  |      Origin        |  State  |||
+||+-----------------------+------------+--------------------+---------+||
+|||  10.10.0.0/16         |  local     |  CreateRouteTable  |  active |||
+||+-----------------------+------------+--------------------+---------+||
+
+
+aws ec2 create-tags \
+--resources rtb-09e96ac9c646cc937  \
+--region ap-southeast-2  \
+--tags Key=Name,Value="devopsacademy-rt-private"
+```
+
+
+- Commands for associating the route tables with subnets:
+```
+aws ec2 create-route --route-table-id rtb-088d967af4f789ea7 \
+--destination-cidr-block 0.0.0.0/0 \
+--gateway-id igw-02e59fc8fcb21e78c \
+--region ap-southeast-2
+
+aws ec2 create-route --route-table-id rtb-09e96ac9c646cc937 \
+--destination-cidr-block 0.0.0.0/0 \
+--gateway-id nat-0854577e398e17703 \
+--region ap-southeast-2
+```

```

Comment:
Those are the answers for the question below `Commands for creating the following routes:`

- You created two route tables (private and public).
- You associated two default gateways (one with which route)

That is missing is associating the routes tables with the subnets you created before.

You need to associate the private route tables with 3 private subnets, then the public route table with 3 public subnets.

That association is necessary otherwise the subnets will use the default route table (that has no gateway)